ethereum/spec/src/main/resources/tech/pegasys/teku/spec/config/presets/minimal/capella.yaml

```diff
@@ -11,4 +11,9 @@ MAX_BLS_TO_EXECUTION_CHANGES: 16
 # Execution
 # ---------------------------------------------------------------
 # 2**2 (= 4)
-MAX_WITHDRAWALS_PER_PAYLOAD: 4
\ No newline at end of file
+MAX_WITHDRAWALS_PER_PAYLOAD: 4
+
+# Withdrawals processing
+# ---------------------------------------------------------------
+# 2**14 (= 16384) validators
+MAX_VALIDATORS_PER_WITHDRAWALS_SWEEP: 16384
```

Comment:
Minimal should set the max to 16.  I'd suggest setting to 16 for swift as well since it usually follows minimal spec.

Reply:
nice catch!